templates/includes/sidebar.html

```diff
@@ -1,4 +1,15 @@
 <ul class="sidebar-nav">
+  {% if user.is_authenticated %}
+    <li>User: <a href="{% url 'taxi:driver-detail' pk=user.id %}">{{ user.username }}</a></li>
```

Comment:
Ensure that the `driver-detail` URL pattern in your `urls.py` accepts a `pk` parameter. If the `user.id` is not the correct identifier for the driver, you may need to adjust this to match the expected parameter in your URL configuration.